runner/ansible/roles/2.2.3/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.2.3'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # Check the pacemaker version IS
+    # If not installed, exit with error
+    rpm -q --qf "%{VERSION}\n" pacemaker || exit 2
```

Comment:
I don't think we need the `exit 2` here.
If the package doesn't exist, the `rpm` command will return a `1`, so I think it is neater to have the `failed_when: config_udpated.rc > 0`.